src/sections/Projects/Sistent/components/divider/code.js

```diff
@@ -0,0 +1,306 @@
+import { useLocation } from "@reach/router";
+import { navigate } from "gatsby";
+import React from "react";
+
+import {
+  Box,
+  Chip,
+  Divider,
+  List,
+  ListItem,
+  ListItemText,
+  SistentThemeProvider,
+  styled,
+} from "@layer5/sistent";
+
+import TabButton from "../../../../../reusecore/Button";
+import { useStyledDarkMode } from "../../../../../theme/app/useStyledDarkMode";
+import { SistentLayout } from "../../sistent-layout";
+
+import FormatBoldIcon from "@mui/icons-material/FormatBold";
+import FormatItalicIcon from "@mui/icons-material/FormatItalic";
+import FormatAlignRightIcon from "@mui/icons-material/FormatAlignRight";
+import FormatAlignCenterIcon from "@mui/icons-material/FormatAlignCenter";
+import FormatAlignLeftIcon from "@mui/icons-material/FormatAlignLeft";
+
+import { CodeBlock } from "../button/code-block";
+
+const style = {
+  py: 0,
+  width: "100%",
+  maxWidth: 360,
+  borderRadius: 2,
+  border: "1px solid",
+  borderColor: "divider",
+  backgroundColor: "background.paper",
+};
+
+const Root = styled("div")(({ theme }) => ({
+  width: "100%",
+  ...theme.typography.body2,
+  color: theme.palette.text.secondary,
+  "& > :not(style) ~ :not(style)": {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+const content = (
+  <p style={{ color: "white" }}>
+    {"Lorem ipsum dolor sit amet, consectetur adipiscing elit."}
+  </p>
+);
+
+const codes = [
+  `<List sx={style}>
+    <ListItem>
+      <ListItemText primary="Full width variant below" />
+    </ListItem>
+    <Divider component="li" />
+    <ListItem>
+      <ListItemText primary="Inset variant below" />
+    </ListItem>
+    <Divider variant="inset" component="li" />
+    <ListItem>
+      <ListItemText primary="Middle variant below" />
+    </ListItem>
```

Comment:
![Screenshot_20241112-093749.png](https://github.com/user-attachments/assets/4074157b-0f58-451c-853f-e04bc9191249)

Can you please align the lorem ipsum lines with the other text?